doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -160,6 +160,17 @@ Zigbee
 
 |no_changes_yet_note|
 
+Gaezll
+------
+
+* Added:
+
+   * User-specific file for Gazell pairing. User can override Gazell pair config inside application.
+
+* Fixed:
+
+   * Clear system address and host id in RAM when ``gzp_erase_pairing_data`` is called.
```

Comment:
The C function style should make it :c:func:`gzp_erase_pairing_data`.

Reply:
The C function style should be :c:func:\`gzp_erase_pairing_data\`. Please search the releases notes for examples.

Reply:
Technical writer has the same comment. Resolve this one.